### DISCUSSION TITLE
开启行存模式，属性不支持？

### DISCUSSION THREAD

**Discussion by lifekevin21** [apache/doris#21743] [A - General / Q&A] [answered]
mysql> CREATE TABLE `tbl_point_query` (
    ->     `key` int(11) NULL,
    ->     `v1` decimal(27, 9) NULL,
    ->     `v2` varchar(30) NULL,
    ->     `v3` varchar(30) NULL,
    ->     `v4` date NULL,
    ->     `v5` datetime NULL,
    ->     `v6` float NULL,
    ->     `v7` datev2 NULL
    -> ) ENGINE=OLAP
    -> UNIQUE KEY(`key`)
    -> COMMENT 'OLAP'
    -> DISTRIBUTED BY HASH(`key`) BUCKETS 1
    -> PROPERTIES (
    ->     "replication_allocation" = "tag.location.default: 1",
    ->     "enable_unique_key_merge_on_write" = "true",
    ->     "light_schema_change" = "true",
    ->     "store_row_column" = "true"
    -> );
ERROR 1105 (HY000): errCode = 2, detailMessage = **Unknown properties: {store_row_column=true}**

**Comment by eldenmoon** [comment, marked as answer]
需要升级到2.0-beta
